src/app/shared/mocks/dspace-rest/endpoint-mocking-rest.service.ts

```diff
@@ -7,6 +7,7 @@ import { RestRequestMethod } from '../../../core/data/rest-request-method';
 import { RawRestResponse } from '../../../core/dspace-rest/raw-rest-response.model';
 import { DspaceRestService, HttpOptions } from '../../../core/dspace-rest/dspace-rest.service';
 import { MOCK_RESPONSE_MAP, ResponseMapMock } from './mocks/response-map.mock';
+// eslint-disable-next-line import/no-namespace
 import * as URL from 'url-parse';
```

Comment:
This import can be left out
The usage below will just work with native `new URL(...)`

Reply:
Good find, @ybnd .  I'll get these usages of url-parse cleaned up and verify they work for me.